Misc/NEWS.d/next/Library/2018-10-19-09-32-07.bpo-35017.vAanVd.rst

```diff
@@ -0,0 +1,2 @@
+After shutdown, :class:`BaseServer` accepted a last request if it was sent
```

Comment:
As a personal suggestion I think this can be reworded to represent the behavior the PR proposes instead of the current behavior like below. I would wait for reviewer feedback though. 

```markdown
:class:`BaseServer` rejects the request sent between polling and polling 
timeout after shutdown.
```

Other than that it looks good to me. Thanks for the PR :)

Reply:
Please mention the module name. Something like:

```
:meth:`socketserver.BaseServer.serve_forever` now exits immediately
if it's :meth:`~socketserver.BaseServer.shutdown` method is called
while it is polling for new events.
```

Reply:
I removed the former news entry and added a new one with your description. I couldn't have described this better.